addons/website/static/src/scss/primary_variables.scss

```diff
@@ -1961,6 +1961,15 @@ $o-base-website-values-palette: (
     'headings-margin-top': 0,
     'headings-margin-bottom': 0.5rem,
 
+    'display-font-sizes': (
+        1: 4.5rem,
+        2: 4rem,
+        3: 3.5rem,
+        4: 3rem,
+        5: 2.5rem, // We don't use it.
+        6: 2rem, // We don't use it.
+    ),
```

Comment:
I'm not 100% sure I understand your point. We want to use bootstrap's `display-N` classes in the website builder. Bootstrap provides a variable to customize the text size of these classes. Yes, it's in the form of a map and this makes reading the variable a little more complicated (but simpler with a regEx I admit), but we don't need to override the `display-N` classes to apply the font size stored in our new variable that does the same thing as the BS one.

Of course, I created variables for the other things we want to parameterize, but I did so because these variables don't exist in BS by default. For font sizes, the variable already exists.

If we want the bootstrap default values, I can do the same as for the other variables defined here `'display-font-sizes': null,`.

As the change you ask here will imply quite a lot of a change in the code i'd rather be sure it's for the better :)

Reply:
I don't think I am asking for something that changes a lot in the code: on the contrary, it will avoid you having to support parsing maps into JSON :wink: 

What I am suggesting is not that you stop using the `$display-font-sizes` variable of bootstrap. It should of course be initialized to the right value, corresponding the display font-sizes. But in the website values "palette", you can just declare the 4 we need without a map. Then in the bootstrap_overridden file, you assign those 4 values inside the boostrap map. Then the rest of the SCSS should still rely on using the bootstrap map *but* your JS can access the 4 values we need without the need of parsing a map into JSON.